### PR TITLE
[SDESK-787] Prefix 'Archived from' to desk information for archived items

### PR DESCRIPTION
**Pull request by nrvikas**
Depends on https://github.com/superdesk/superdesk-core/pull/816

### CODE REVIEW

**Comment by nrvikas** [comment]
Thanks @petrjasek , added the annotations.

**Comment by nrvikas** [comment]
Sure @petrjasek , sorry I missed it before. The string gets auto converted to upper case because of the 'stage' class. So, that's taken care.

**Comment by nrvikas** [comment]
Sure @petrjasek , sorry I missed it before. It is auto converting to uppercase because of 'stage' class included. For the react part, I have used gettextCatalog.getString('Archived From '). I hope the empty space in the end will not result in setting a new string for translation. If it's a problem, pls let me know, I will change it.

File: scripts/apps/search/components/ItemContainer.jsx [hunk 1]
```diff
@@ -10,7 +11,7 @@ export function ItemContainer(props) {
     if (item._type !== 'ingest') {
         if (desk) {
             label = gettext('desk:');
-            value = desk.name;
+            value = item._type !== 'archived' ? desk.name : gettextCatalog.getString('Archived From ') + desk.name;
```
Comment (mdhaman) [inline scripts/apps/search/components/ItemContainer.jsx]: @nrvikas change this to `Archived from` as well.

Comment (nrvikas) [inline scripts/apps/search/components/ItemContainer.jsx]: Done this @mdhaman . Tks.

File: scripts/apps/search/components/ItemContainer.jsx [hunk 2]
```diff
@@ -10,7 +11,7 @@ export function ItemContainer(props) {
     if (item._type !== 'ingest') {
         if (desk) {
             label = gettext('desk:');
-            value = desk.name;
+            value = item._type !== 'archived' ? desk.name : gettextCatalog.getString('Archived from ') + desk.name;
```
Comment (petrjasek) [inline scripts/apps/search/components/ItemContainer.jsx]: there is still an extra space in the string `'Archived from '`

Comment (nrvikas) [inline scripts/apps/search/components/ItemContainer.jsx]: Done Petr :D

**Comment by mdhaman** [comment]
@petrjasek this commit is not merged into 1.6. Can you please merge this commit into 1.6?

**Comment by petrjasek** [comment]
@mdhaman done

**Comment by mdhaman** [comment]
@petrjasek thanks

**Pull request merged**